features/org.wso2.carbon.appmgt.store.feature/src/main/resources/store/extensions/assets/webapp/pages/myapps.jag

```diff
@@ -149,17 +149,52 @@ require('/modules/store.js').exec(function (ctx) {
             options.lifecycleState = ["published"];
             assets = store.taggeds(type, options, paging);
         } else if (query) { // if search from search box
-            urlQuery = 'query=' + query + '&';
-            query = require('/modules/search.js').build(query);
-            query.overview_treatAsASite = 'FALSE';
-            if (!isMultipleVersionDisplayEnabled) {
-                query.overview_makeAsDefaultVersion = 'true';
+            var searchParameter;
+            var searchValue;
+            var splitQuery = query.split(":");
+            searchParameter = splitQuery[0];
+            // This is a registry based search. In search box, user wants to search by business owner name. But
+            // in registry, there is only business owner Id. Therefore, it is necessary to get business owner Ids
+            // related to search prefix from DB.
+            if (!searchParameter.equals("Business Owner")) {
+                urlQuery = 'query=' + query + '&';
+                query = require('/modules/search.js').build(query);
+                query.overview_treatAsASite = 'FALSE';
+                if (!isMultipleVersionDisplayEnabled) {
```

Comment:
Logic is bit unclear. 

Business owner can be assigned to both sites/webapps.
Why are we setting overview_treatAsASite = 'FALSE' without checking the registry artifact property?

And isMultipleVersionDisplayEnabled is a global property not bind to an artifact. Why are we setting "overview_makeAsDefaultVersion = 'true' " considering the global setting?